apps/game_backend/lib/game_backend/quests/quest.ex

```diff
@@ -19,6 +20,10 @@ defmodule GameBackend.Quests.Quest do
     timestamps()
   end
 
+  @types ["daily", "bounty", "weekly"]
+
+  def types(), do: @types
```

Comment:
Modify `field(:type)` to be `field(:type, Ecto.Enum,  values: [:daily, :bounty, :weekly])`

Then if you need the possible values you can do `Ecto.Enum.values(GameBackend.Quests.Quest, :type)`

Bonus, you can remove `validate_inclusion/3` below

Reply:
Nice suggestion! done